tower/src/ready_cache/error.rs

```diff
@@ -6,9 +6,12 @@ pub struct Failed<K>(pub K, pub crate::BoxError);
 
 // === Failed ===
 
-impl<K> std::fmt::Debug for Failed<K> {
+impl<K: std::fmt::Debug> std::fmt::Debug for Failed<K> {
```

Comment:
Note that this is a backwards-incompatible change. So would have to be slated for next major release (which I guess we're in the process of preparing probably). cc @LucioFranco

Reply:
we are, in fact (and linkerd already depends on tower master)

Reply:
👍 I am hoping to hop back on after tokio 0.3 so fine to push this onto master